<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newTaskDialog</class>
 <widget class="QDialog" name="newTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>741</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>单位信息</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>741</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>85</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>处置信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>542</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>保存位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>472</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>事件描述</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>73</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="comLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="principalLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="officePhoneLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="systemNameLineEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="eventSourceComboBox">
      <item>
       <property name="text">
        <string>部级发现</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>市局发现</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>市局直属单位及分县局发现</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他单位发现</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>72</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="comAddrLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneLineEdi"/>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="sysLevelComboBox">
      <item>
       <property name="text">
        <string>无</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>一级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>二级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>三级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>四级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>五级</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="discribeTextEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请简单描述事件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="savePlaceLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>540</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pathButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>537</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>自定义</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>391</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>355</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>389</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>425</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>354</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="eventKindComboBox">
      <item>
       <property name="text">
        <string>攻击事件</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>设备设施故障</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>灾害性事件</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="disposeComComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLineEdit" name="eventNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="eventPlaceLineEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="eventLevelComboBox">
      <item>
       <property name="text">
        <string>一般事件(IV级)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>较大事件(III级)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>重大事件(II级)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>特大事件(I级)</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>426</y>
     <width>8</width>
     <height>8</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/icon/image/asterisk.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>81</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事发单位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>118</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>负责人</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>152</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>办公电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>187</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>系统名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>223</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事件来源</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>115</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>联系电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>80</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>单位地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>149</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电子邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>433</x>
     <y>184</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>系统等级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>350</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事件名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>384</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事发地点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>420</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事件级别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>350</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事发时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>422</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>处置单位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>386</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>事件类型</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>newTaskDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>553</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
